libs/azure-ai/langchain_azure_ai/callbacks/tracers/inference_tracing.py

```diff
@@ -114,6 +114,16 @@ def _safe_json(obj: Any) -> str:
         return '"<unserializable>"'
 
 
+def _try_parse_json(s: Any) -> Any:
+    """Best-effort parse a JSON string, otherwise return the original value."""
+    if isinstance(s, str):
```

Comment:
Since `json.loads` takes str, bytes or bytearray, this should be more permissive to allow a byte string

```suggestion
    if isinstance(s, (str, bytes)):
```

Reply:
thanks!